.gitignore

```diff
@@ -2,3 +2,4 @@
 __pycache__
 logs.log
 geckodriver.log
+Driver/
```

Comment:
You can add comments here specifying what some of these files are and if necessary the reason why they need to be ignored [Suggestion]

Reply:
sure